data/targets.yml

```diff
@@ -13,6 +23,16 @@
   obs_project: "Devel:YaST:Head"
   obs_sr_project: "SUSE:SLE-12-SP2:GA"
   obs_target: "SLE-12-SP1"
+:sle12sp3:
+  obs_api: "https://api.suse.de/"
+  obs_project: "Devel:YaST:Head"
+  obs_sr_project: "SUSE:SLE-12-SP3:GA"
+  obs_target: "sle12_sp3"
+:sle_latest:
```

Comment:
NP: I was just thinking about a reference in the YAML to share the same part:

```yaml
:sle12sp3: &1
  obs_api: "https://api.suse.de/"
  ...
:sle_latest: *1
```

But that would be probably less readable...